AndroidBuildPlugin.excludeKnownFiles excludes Kotlin reflection

```
java.lang.ExceptionInInitializerError
at kotlin.reflect.jvm.internal.impl.types.error.ErrorModuleDescriptor.<clinit>(ErrorModuleDescriptor.kt:23)
at kotlin.reflect.jvm.internal.impl.types.error.ErrorUtils.<clinit>(ErrorUtils.kt:22)
at kotlin.reflect.jvm.internal.impl.types.error.ErrorUtils.createErrorType(ErrorUtils.kt)
at kotlin.reflect.jvm.internal.impl.types.TypeUtils.<clinit>(TypeUtils.java:36)
at kotlin.reflect.jvm.internal.impl.descriptors.impl.AbstractClassDescriptor$1.invoke(AbstractClassDescriptor.java:49)
at kotlin.reflect.jvm.internal.impl.descriptors.impl.AbstractClassDescriptor$1.invoke(AbstractClassDescriptor.java:46)
at kotlin.reflect.jvm.internal.impl.storage.LockBasedStorageManager$LockBasedLazyValue.invoke(LockBasedStorageManager.java:408)
at kotlin.reflect.jvm.internal.impl.storage.LockBasedStorageManager$LockBasedNotNullLazyValue.invoke(LockBasedStorageManager.java:527)
at kotlin.reflect.jvm.internal.impl.descriptors.impl.AbstractClassDescriptor.getDefaultType(AbstractClassDescriptor.java:175)
at kotlin.reflect.jvm.internal.impl.builtins.jvm.JvmBuiltInsCustomizer.createMockJavaIoSerializableType(JvmBuiltInsCustomizer.kt:91)
at kotlin.reflect.jvm.internal.impl.builtins.jvm.JvmBuiltInsCustomizer.<init>(JvmBuiltInsCustomizer.kt:59)
at kotlin.reflect.jvm.internal.impl.builtins.jvm.JvmBuiltIns$customizer$2.invoke(JvmBuiltIns.kt:76)
at kotlin.reflect.jvm.internal.impl.builtins.jvm.JvmBuiltIns$customizer$2.invoke(JvmBuiltIns.kt:75)
at kotlin.reflect.jvm.internal.impl.storage.LockBasedStorageManager$LockBasedLazyValue.invoke(LockBasedStorageManager.java:408)
at kotlin.reflect.jvm.internal.impl.storage.LockBasedStorageManager$LockBasedNotNullLazyValue.invoke(LockBasedStorageManager.java:527)
at kotlin.reflect.jvm.internal.impl.storage.StorageKt.getValue(storage.kt:42)
at kotlin.reflect.jvm.internal.impl.builtins.jvm.JvmBuiltIns.getCustomizer(JvmBuiltIns.kt:75)
at kotlin.reflect.jvm.internal.impl.load.kotlin.DeserializationComponentsForJava.<init>(DeserializationComponentsForJava.kt:80)
at kotlin.reflect.jvm.internal.impl.load.kotlin.DeserializationComponentsForJavaKt.makeDeserializationComponentsForJava(DeserializationComponentsForJava.kt:192)
at kotlin.reflect.jvm.internal.impl.load.kotlin.DeserializationComponentsForJava$Companion.createModuleData(DeserializationComponentsForJava.kt:123)
at kotlin.reflect.jvm.internal.impl.descriptors.runtime.components.RuntimeModuleData$Companion.create(RuntimeModuleData.kt:32)
at kotlin.reflect.jvm.internal.ModuleByClassLoaderKt.getOrCreateModule(moduleByClassLoader.kt:58)
at kotlin.reflect.jvm.internal.KDeclarationContainerImpl$Data$moduleData$2.invoke(KDeclarationContainerImpl.kt:36)
at kotlin.reflect.jvm.internal.KDeclarationContainerImpl$Data$moduleData$2.invoke(KDeclarationContainerImpl.kt:35)
at kotlin.reflect.jvm.internal.ReflectProperties$LazySoftVal.invoke(ReflectProperties.java:93)
at kotlin.reflect.jvm.internal.ReflectProperties$Val.getValue(ReflectProperties.java:32)
at kotlin.reflect.jvm.internal.KDeclarationContainerImpl$Data.getModuleData(KDeclarationContainerImpl.kt:35)
at kotlin.reflect.jvm.internal.KClassImpl$Data$descriptor$2.invoke(KClassImpl.kt:50)
at kotlin.reflect.jvm.internal.KClassImpl$Data$descriptor$2.invoke(KClassImpl.kt:48)
at kotlin.reflect.jvm.internal.ReflectProperties$LazySoftVal.invoke(ReflectProperties.java:93)
at kotlin.reflect.jvm.internal.ReflectProperties$Val.getValue(ReflectProperties.java:32)
at kotlin.reflect.jvm.internal.KClassImpl$Data.getDescriptor(KClassImpl.kt:48)
at kotlin.reflect.jvm.internal.KClassImpl.getDescriptor(KClassImpl.kt:182)
at kotlin.reflect.jvm.internal.KClassImpl.getMemberScope$kotlin_reflection(KClassImpl.kt:191)
at kotlin.reflect.jvm.internal.KClassImpl.getProperties(KClassImpl.kt:207)
at kotlin.reflect.jvm.internal.KDeclarationContainerImpl.findPropertyDescriptor(KDeclarationContainerImpl.kt:80)
at kotlin.reflect.jvm.internal.KPropertyImpl$_descriptor$1.invoke(KPropertyImpl.kt:125)
at kotlin.reflect.jvm.internal.KPropertyImpl$_descriptor$1.invoke(KPropertyImpl.kt:124)
at kotlin.reflect.jvm.internal.ReflectProperties$LazySoftVal.invoke(ReflectProperties.java:93)
at kotlin.reflect.jvm.internal.KPropertyImpl.getDescriptor(KPropertyImpl.kt:128)
at kotlin.reflect.jvm.internal.KPropertyImpl.computeDelegateSource(KPropertyImpl.kt:81)
at kotlin.reflect.jvm.internal.KProperty0Impl$delegateValue$1.invoke(KProperty0Impl.kt:37)
at kotlin.SafePublicationLazyImpl.getValue(LazyJVM.kt:107)
at kotlin.reflect.jvm.internal.KProperty0Impl.getDelegate(KProperty0Impl.kt:39)
at kotlin.jvm.internal.PropertyReference0.getDelegate(PropertyReference0.java:46)
at net.twisterrob.android.viewbinding.ViewBindingsTest.testViewBindingFragmentDestroyView$lambda$12$lambda$9(ViewBindingsTest.kt:179)
at net.twisterrob.android.viewbinding.ViewBindingsTest.$r8$lambda$TRXnVjBQiPjhxWeFjbtNzcMYdQA(ViewBindingsTest.kt)
at net.twisterrob.android.viewbinding.ViewBindingsTest$$ExternalSyntheticLambda0.perform(D8$$SyntheticClass)
at androidx.fragment.app.testing.FragmentScenario.onFragment$lambda$6(FragmentScenario.kt:323)
at androidx.fragment.app.testing.FragmentScenario.$r8$lambda$wBwdZQK-bWVeBF0zvDC9UI80cxs(FragmentScenario.kt)
at androidx.fragment.app.testing.FragmentScenario$$ExternalSyntheticLambda2.perform(D8$$SyntheticClass)
at androidx.test.core.app.ActivityScenario.lambda$onActivity$2$androidx-test-core-app-ActivityScenario(ActivityScenario.java:789)
at androidx.test.core.app.ActivityScenario$$ExternalSyntheticLambda2.run(D8$$SyntheticClass)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
at android.app.Instrumentation$SyncRunnable.run(Instrumentation.java:1855)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5221)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)
Caused by: java.lang.IllegalStateException: Resource not found in classpath: kotlin/kotlin.kotlin_builtins
at kotlin.reflect.jvm.internal.impl.serialization.deserialization.builtins.BuiltInsLoaderImpl.createBuiltInPackageFragmentProvider(BuiltInsLoaderImpl.kt:59)
at kotlin.reflect.jvm.internal.impl.serialization.deserialization.builtins.BuiltInsLoaderImpl.createPackageFragmentProvider(BuiltInsLoaderImpl.kt:35)
at kotlin.reflect.jvm.internal.impl.builtins.KotlinBuiltIns.createBuiltInsModule(KotlinBuiltIns.java:105)
at kotlin.reflect.jvm.internal.impl.builtins.DefaultBuiltIns.<init>(DefaultBuiltIns.kt:24)
at kotlin.reflect.jvm.internal.impl.builtins.DefaultBuiltIns.<init>(DefaultBuiltIns.kt:21)
at kotlin.reflect.jvm.internal.impl.builtins.DefaultBuiltIns.<clinit>(DefaultBuiltIns.kt:31)
... 64 more
```

--- a/plugin/building/src/main/kotlin/net/twisterrob/gradle/android/AndroidBuildPlugin.kt
+++ b/plugin/building/src/main/kotlin/net/twisterrob/gradle/android/AndroidBuildPlugin.kt
@@ -151,10 +151,12 @@ class AndroidBuildPlugin : net.twisterrob.gradle.common.BasePlugin() {
 				"META-INF/androidx.*.version",
 
 				// Needed for compiling against top-level functions. Since APK is end product, this is not necessary.
+				// TODO consider not including this for AARs.
 				"**/*.kotlin_metadata",
-				// Kotlin builds these things in, found no docs so far about their necessity, so try to exclude.
-				"**/*.kotlin_module",
-				"**/*.kotlin_builtins",
+				// kotlin_module file holds @OptionalExpectation, @JvmMultifileClass, module annotations.
+				//"**/*.kotlin_module",
+				// kotlin_builtins are required for org.jetbrains.kotlin:kotlin-reflect.
+				//"**/*.kotlin_builtins",
 
 				// Readmes
 				// (e.g. hamcrest-library-2.1.jar and hamcrest-core-2.1.jar both pack a readme to encourage upgrade)

